phpstan.neon

```diff
@@ -7,8 +7,6 @@ parameters:
     ignoreErrors:
         # To be fixed:
         - '#Call to an undefined method Facebook\\WebDriver\\WebDriver::getTouch\(\)#'
-        - '#Call to an undefined method Facebook\\WebDriver\\WebDriverElement::getCoordinates\(\)#'
-        - '#Call to an undefined method Facebook\\WebDriver\\WebDriverElement::equals\(\)#'
```

Comment:
And now those needs to be put back :upside_down_face: .

Reply:
Thank you !
Happy you did point that out because I was searching why the error was occurring ^^